netty/src/main/java/io/grpc/netty/NettyClientTransport.java

```diff
@@ -244,7 +250,9 @@ private void notifyTerminated(Status status) {
 
   private NettyClientHandler newHandler() {
     Http2Connection connection = new DefaultHttp2Connection(false);
-    Http2FrameReader frameReader = new DefaultHttp2FrameReader();
+    Http2HeadersDecoder headersDecoder =
+        new DefaultHttp2HeadersDecoder(maxHeaderListSize, Http2CodecUtil.DEFAULT_HEADER_TABLE_SIZE);
```

Comment:
Here the header table size is not configurable in gRPC, so we shouldn't have a redundant constant for it.